torch/linalg/__init__.py

```diff
@@ -309,26 +347,25 @@
 eigvals = _add_docstr(_linalg.linalg_eigvals, r"""
 linalg.eigvals(input, *, out=None) -> Tensor
 
-Computes the eigenvalues of a square
-matrix :attr:`input`, or of each such matrix in a batched :attr:`input`.
-
-The eigenvalues are not necessarily sorted.
-For real-valued input the resulting eigenvalues will always be of complex type.
+Computes the eigenvalues of a square matrix or a batch of matrices :attr:`input`.
 
-Supports input of float, double, cfloat and cdouble dtypes.
+Supports :attr:`input` of float, double, cfloat and cdouble dtypes.
+The eigenvalues will always be complex-valued, even if :attr:`input` is real.
```

Comment:
See previous notes about clarifying mathematical statements vs. statements about returned tensors and placement of sentences like this.